README.md

```diff
@@ -109,11 +78,70 @@ s3://mybucket/aws-instance-scheduler/v1.4.0/instance-scheduler.template
 s3://mybucket/aws-instance-scheduler/v1.4.0/instance-scheduler-remote.template
 ```
 
-## Deploy
+### Deploy
 
 See the [AWS Instance Scheduler Implementation Guide](https://s3.amazonaws.com/solutions-reference/aws-instance-scheduler/latest/instance-scheduler.pdf) for deployment instructions, using the link to the instance-scheduler.template from your bucket, rather than the one for AWS Solutions. Ex. https://mybucket.s3.amazonaws.com/aws-instance-scheduler/v1.4.0.mybuild/instance-scheduler.template
 
-## CDK Documentation
+
+# Testing the Solution
+
+## Running Tests Locally
+to test the solution you will need to install tox
+
+```pip install tox```
+
+Then from the root directory of the solution
```

Comment:
elaborate a little more, maybe only suggest running all the tests unless you also want to explain when and why to only run one set of tests.

Reply:
the extra commands are just technical documentation. If you are trying to run the tests yourself I would expect you to know if you want to run all tests or just a subset (i.e. have been making changes to specifically the lambda or the cdk code)

Reply:
But this is a readme. It should be high-level. If I don't know what commands to run, I'd probably be looking at the readme, and this doesn't help me figure that out. If I am looking for technical documentation, I don't think I'd go straight to the readme first, but if I did, this still doesn't help me figure that out.